spec/fabricators/user_fabricator.rb

```diff
@@ -4,17 +4,21 @@
 
 Fabricator(:user, class_name: :user) do
   transient refresh_auto_groups: false
+  transient trust_level: nil
 
   name "Bruce Wayne"
   username { sequence(:username) { |i| "bruce#{i}" } }
   email { sequence(:email) { |i| "bruce#{i}@wayne.com" } }
   password "myawesomepassword"
-  trust_level TrustLevel[1]
   ip_address { sequence(:ip_address) { |i| "99.232.23.#{i % 254}" } }
   active true
 
+  after_build { |user, transients| user.trust_level = transients[:trust_level] || TrustLevel[1] }
```

Comment:
From documentation I couldn't find another way to figure out if `trust_level` was excplicitly passed or not. If there's a nicer way, I'm all ears. 🙂

Reply:
Yeah I think this is about right :+1: